helper_dart/test/user_token_storage_test.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:algolia_helper/src/user_token_storage.dart';
+import 'package:test/test.dart';
+
+void main() {
+
+  test('check user token persistent storage', () async {
+    final userTokenController = UserTokenStorage()
```

Comment:
here also an instance of the storage class is called a controller. Seems odd to me